src/components/mortgage-investment-compare/index.js

```diff
@@ -22,100 +22,86 @@ import {
 
 const COMPOUND_FREQUENCY = 12
 
-const DEFAULT_OPTION = {
-  interestAmt: 0,
-  interestRate: 0,
-  term: 0,
-  pmt: 0,
-  fv: 0
-}
-
-let option1 = _.cloneDeep(DEFAULT_OPTION)
-let option2 = _.cloneDeep(DEFAULT_OPTION)
-
-option1.interestRate = 4
-option1.term = 15
-
-option2.interestRate = 4.3
-option2.term = 30
-
 class MortgageInvestmentCompare extends React.Component {
   constructor () {
     super()
-    this.toggleShowTable = this.toggleShowTable.bind(this)
-    this.calculatePMT = this.calculatePMT.bind(this)
-    this.calculateInterestAmt = this.calculateInterestAmt.bind(this)
     this.calculateFV = this.calculateFV.bind(this)
+    this.calculateInterestAmt = this.calculateInterestAmt.bind(this)
     this.calculateOpportunityCost = this.calculateOpportunityCost.bind(this)
+    this.calculatePMT = this.calculatePMT.bind(this)
+    this.getResult = this.getResult.bind(this)
+    this.toggleShowTable = this.toggleShowTable.bind(this)
     this.updateInput = this.updateInput.bind(this)
-    this.calculateAll = this.calculateAll.bind(this)
-    this.state = {
+
+    const input = {
+      inflation: 2,
       investmentRate: 9,
       loanAmt: 200000,
-      inflation: 2,
-      optCost: 0,
-      options: {
-        option1,
-        option2
+      option1: {
+        interestRate: 4,
+        term: 15
       },
-      shorterOption: '',
-      longerOption: '',
-      yearlyResultsByOption: [],
-      showTable: false
```

Comment:
why remove these? It's always a good practice to have an initial state in React, to remind you of all the props.